packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -456,10 +456,11 @@ const Conversation = WebexPlugin.extend({
   * Gets an array of activities with an array of activity URLS
   * @param {Array} activityUrls
   * @param {String} [cluster] cluster where the activities are located
+  * @param {String} url base convo url where the activities are located
   * @returns {Promise<Object>} Resolves with the activities
   * TODO: add cluster functionality when clusters are ready
   */
-  bulkActivitiesFetch(activityUrls, cluster) {
+  bulkActivitiesFetch(activityUrls, cluster, url) {
```

Comment:
```suggestion
  bulkActivitiesFetch(activityUrls, {cluster, url}) {
```
I like this better since you wouldn't be passing in `cluster` and `url` at the same time.
It would be a breaking change.

Reply:
```suggestion
  bulkActivitiesFetch(activityUrls, option) {
    let cluster, url;
    if (typeof option === 'string') {
      cluster = option;
    }
    else {
      ({cluster, url}= options);
    }
```
If we don't want to have a breaking change.

Reply:
agree, i think `options` is better since you wouldn't define both cluster and url at the same time.

Reply:
@aimeex3, would you be up for making this change, appears it's not blocking, but would like to confirm before we proceed with a rebase and merge.

Reply:
yeah i can make the change

Reply:
@InteractiveTimmy updated! good to merge after checks are completed